demo/datayoga/jobs/receiver.yaml

```diff
@@ -0,0 +1,23 @@
+input:
+
+  uses: http.receiver
+  with:
+    host: 0.0.0.0
+    port: 8080
+
+steps:
+
+  - uses: map
+    with:
+      expression:
+        id: id
+        fname: first_name
+        lname: last_name
+        email: email
+      language: jmespath
+
```

Comment:
I'd remove empty lines unless you think they are helpful somehow.

Reply:
It looks more readable.